x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style_defaults.js

```diff
@@ -158,7 +172,7 @@ export function getDefaultDynamicProperties() {
       },
     },
     [VECTOR_STYLES.ICON_ORIENTATION]: {
-      type: VectorStyle.STYLE_TYPE.STATIC,
```

Comment:
Not entirely related to this PR, but do we really want to change this when clients of `getDefaultDynamicProperties` are not going to fill in the missing field parameter? It's unused code. I would almost just remove ICON_ORIENTATION, LABEL_TEXT, ... from this return-value, or change signature to `getDefaultDynamicPropertiesForPropertyType(propertyType)` and return each property-type individually.

Reply:
Lets move this discussion to a separate issue. This PR was just fixing an error where the defaults were incorrectly labeled as static.